### DISCUSSION TITLE
Clarifying on "Check if third-party domains comply with EFF's Do Not Track policy" in General Settings

### DISCUSSION THREAD

**Discussion by SpookyYomo** [EFForg/privacybadger#2965] [Q&A] [answered]
On Firefox with Privacy Badger installed as an extension, there is a setting "Check if third-party domains comply with EFF's Do Not Track policy" present in the General settings window.
I am trying to understand what it does, particularly because I understand that sending DNT to sites that do not respect it only increases tracking, and am hoping that this setting means that a DNT is not sent to websites that do not respect DNT.

**Comment by ghostwords** [comment]
Hello!

Privacy Badger sends Do Not Track and [Global Privacy Control](https://globalprivacycontrol.org/) signals in order to tell websites that you do not wish to be tracked (DNT) and that you would like to opt out of having your data shared or sold (GPC). We currently believe the privacy benefits (signalling intent with potential legal implications) outweighs the privacy cost (1 bit of information per signal).

You may be interested in #2950.

**Reply by SpookyYomo** [reply]
Hi, thanks for the reply! :) I understand from <coveryourtracks.eff.org> (I don't recall the URL off the top of my head) as DNT being 1 bit of information these days yes, and so it's a fairly small cost.
Regardless, do I interpret that if the extension setting 'Send websites the "[Global Privacy Control](https://globalprivacycontrol.org/)" and "[Do Not Track](https://www.eff.org/issues/do-not-track)" signals'  is enabled, regardless of what "Check if third-party domains comply with EFF's Do Not Track policy" is, then a DNT signal is always sent?

**Reply by ghostwords** [reply, marked as answer]
That's correct.

**Reply by ghostwords** [reply]
With the exception of sites you've disabled Privacy Badger on. When Privacy Badger is disabled for a site, no signals are sent on that site.

**Reply by SpookyYomo** [reply]
Alright! Thanks so much! :D

**Comment by sillyjaybird** [comment]
@ghostwords Can you consider creating separate settings for Global Policy Control and DNT? The reason I ask is that the Brave browser team worked to develop GPC and considers it to be privacy-friendly but not DNT.

 "Second, similar consent mechanisms being pushed by the AdTech industry are incompatible with the kinds of privacy protections which Brave includes. For example, the Orwellian-named “consent management platforms” often require your browser to execute code from tracking companies, just to ask them to not track you. Brave blocks these network requests to tracking companies, which the tracking companies often interpret as “no preference expressed yet”.
[https://brave.com/web-standards-at-brave/4-global-privacy-control/](url)
See also: [https://www.theverge.com/2021/1/28/22252935/global-privacy-control-personal-data-tracking-ccpa-cpra-gdpr-duckduckgo](url)

**Reply by ghostwords** [reply]
I don't see anything in that Brave link about DNT? The Verge link just says that DNT failed, to which I say, yes and no. I suggest subscribing to https://github.com/EFForg/privacybadger/issues/2950.

**Reply by sillyjaybird** [reply]
I've read that and the author clearly thinks DNT is a failure, or at least bad enough that he has little faith in it. And what about users in non-EU countries? Shouldn't they have a chance to opt-out of DNT? I see the value in the GPC but not DNT.
"This ruling is progress, but we need much bolder reforms."

And my point is that having 2 separate settings allows users to choose one or both options.

**Reply by ghostwords** [reply]
Check out the news link in #2950. It appears that DNT is not quite dead.

You could disable GPC/DNT in Privacy Badger and enable GPC using the browser itself (Firefox supports enabling GPC I think) or using another [GPC-capable browser extension](https://globalprivacycontrol.org/orgs#Business).
